README.md

```diff
@@ -314,6 +314,34 @@ To validate prices and financial aid discounts for all programs run:
 
     docker-compose run web ./manage.py validate_db
 
+#### Selenium
+
+To run selenium tests you will first need to create the Javascript bundles.
+Shut down the `watch` docker container if it's running to prevent
+overwriting `webpack-stats.json`:
+
+    docker-compose stop watch
+
+Then, if you are **not** running OS X:
+
+    docker-compose run -e NODE_ENV=prod watch ./webpack_if_prod.sh
```

Comment:
I typically run this directly, by doing something like:

```
drunw node node_modules/webpack/bin/webpack.js --config webpack.config.prod.js --progress --bail
```

the `--progress` option makes the experience slightly nicer, what do you think? (`drunw` is my alias for `docker-compose run watch`)